xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -669,12 +670,6 @@ void CRenderManager::CreateRenderer()
       m_pRenderer = new CRendererMediaCodec;
 #endif
     }
-    if (m_format == RENDER_FMT_MMAL)
```

Comment:
ok, so similar to dxva right? I would keep those lines to indicate that this format is a hw decoder/renderer pair.